scheduler/src/cook/components.clj

```diff
@@ -291,6 +292,13 @@
                                   offer-incubate-ms)
      :mesos-master (fnk [[:config [:mesos master]]]
                         master)
+
+     :mesos-master-hosts (fnk [[:config [:mesos master {master-hosts nil}]]]
```

Comment:
If this is supposed to be configurable, it should be exposed in the end-user config.